pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -55,37 +61,62 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="mesos"></a>Mesos Permissions
 
+The required permissions must be assigned to the principal that issues the
+command to Mesos. This is not always the DC/OS user that is logged into the UI
+or CLI. For example, when a DC/OS user uses the CLI or UI to launch a Marathon
+application the Marathon framework performs its own authorization of that user
+before deciding to create the application. Once Marathon has determined that
+the DC/OS user is permitted to create the application, Marathon uses its own
+DC/OS service account user called `dcos_marathon` to instruct Mesos to launch
+the actual Mesos tasks. At that point, Mesos will check that the
+`dcos_marathon` service account (and not the end user) is authorized to perform
```

Comment:
❓ Maybe clearer if you use "DC/OS user" rather than "end user".

Reply:
This clearly shows my struggle to name things consistently in our docs.

Reply:
"

"Mesos will check that the `dcos_marathon` service user account (and not the DC/OS user account that the HTTP request issued to Marathon is authenticated with) is authorized ..."?

Reply:
@gpaul please can you reply to @jgehrcke 's suggestion. I'd still be OK at this point to ship as-is as it is clearly an improvement and this doc is ripe for conflicts.